lighthouse-core/config/default.js

```diff
@@ -22,15 +22,16 @@ module.exports = {
     "useThrottling": false,
     "gatherers": [
       "service-worker",
-      "offline"
+      "offline",
     ]
   },
   {
     "passName": "redirectPass",
     "useThrottling": false,
     "gatherers": [
       "http-redirect",
-      "html-without-javascript"
+      "html-without-javascript",
+      "start-url",
```

Comment:
we think it actually could live in the offlinePass. order of all the afterPasses should work!

plus nicer for people that only do PWA stuff. just 2 passes instead of 3.